python/paddle/fluid/tests/unittests/parallel_executor_test_base.py

```diff
@@ -68,14 +67,11 @@ def run_executor(exe, binary, feed, fetch_list):
             loss = method(use_feed=feed_dict is not None)
             # NOTE(zjl): memory_optimize/inplace pass would not require 
             # that loss.persistable = True 
-            loss.persistable = memory_opt
+            loss.persistable = False
```

Comment:
Why do you set `False` here?

Reply:
Because all memory optimization strategies do not need `loss.persistable = True`. I set `loss.persistable = False` intentionally to test whether the memory optimization strategy is right.